ui/src/app/utils/index.tsx

```diff
@@ -565,6 +565,10 @@ export const useToggle = (): [boolean, Function] => {
   return [state, setState];
 };
 
+export const useOnMount = (fn: Function) => {
```

Comment:
opt: I think I'd prefer to just use the React library directly in this case. `useOnMount` is more semantically understandable, but it adds a layer redirection which is not portable outside of our immediate codebase, and doesn't really get around the reader needing to know what useEffect does.